dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/fields/FieldsPathParamsHelper.java

```diff
@@ -42,35 +41,30 @@
  * based on {@link FieldFilterParser }. It follows the principles of
  * {@link org.hisp.dhis.fieldfiltering.FieldFilterService}
  */
-public interface InstanceFieldsParamMapper
+public class FieldsPathParamsHelper
```

Comment:
Since you created a fields package this can be package private as its only used by the 2 mapping classes 😄